### PR TITLE
Add section on message validation for mysensors

### PR DESCRIPTION
**Pull request by MartinHjelmare**
**Description:**
Explain message validation for mysensors.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):**
https://github.com/home-assistant/home-assistant/pull/9069

### CODE REVIEW

File: source/_components/mysensors.markdown
Comment (frenck) [inline source/_components/mysensors.markdown]: *Grammar*:
- Missing comma after `validation`: `If a message doesn't pass validation, it will be dropped`

File: source/_components/mysensors.markdown
Comment (frenck) [inline source/_components/mysensors.markdown]: *Spelling & grammar*:
- `dosn't` -> `doesn't`
- Missing comma after `received`: `for a platform has been received, but other required value types are missing`

**Review by frenck — CHANGES_REQUESTED** [review]
⚠️ Found some spelling & grammar issues in the added text.

**Comment by MartinHjelmare** [comment]
Thanks! Should be fixed now.

**Review by frenck — APPROVED** [review]
✅ Thanks for fixing 👍

**Review by Landrash — APPROVED** [review]
Can be merged after parent Pr has been merged!
Thank you :cookie:

**Pull request merged**
